React/Modules/RCTUIManager.m

```diff
@@ -287,7 +290,7 @@ - (void)invalidate
     self->_shadowViewRegistry = nil;
     self->_viewRegistry = nil;
     self->_bridgeTransactionListeners = nil;
-    self->_bridge = nil;
+    [self setBridge:nil];
```

Comment:
This looks suspicious actually. I think this should probably just be a call to super?